rig/machine_control/machine_controller.py

```diff
@@ -199,27 +198,18 @@ def _send_scp(self, x, y, p, *args, **kwargs):
         connection = self._get_connection(x, y)
         return connection.send_scp(length, x, y, p, *args, **kwargs)
 
-    def boot(self, width, height, **boot_kwargs):
+    def boot(self, width, height, force=False, check=True, **boot_kwargs):
```

Comment:
I'm not sure `force` is a good parameter name, to me it implies that a machine can be forced to boot.